From 02413ee2bcb6ffb4f06c55729f39bd3a87766ddf Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 7 Mar 2013 16:03:15 +0800
Subject: [PATCH 388/505] ENGR00253251 mxc v4l2 output:Correct framebuffer
 device name

This patch corrects framebuffer device name which was set
wrongly for Android building environment and non-Android
building environment.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 14788e4..f062e7e 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -95,9 +95,9 @@ void fb_setup(void)
 	int fd;
 
 #ifdef BUILD_FOR_ANDROID
-	fd = open("/dev/fb0",O_RDWR);
-#else
 	fd = open("/dev/graphics/fb0",O_RDWR);
+#else
+	fd = open("/dev/fb0",O_RDWR);
 #endif
 
         alpha.alpha = 0;
-- 
1.8.0

